cirq-ionq/cirq_ionq/ionq_native_target_gateset_test.py

```diff
@@ -0,0 +1,712 @@
+            ),
+        ),
+        (
+            [0, 1],
+            cirq.Circuit(
+                cirq.SingleQubitCliffordGate.Y_sqrt(qubit1[0]),
+                cirq.SingleQubitCliffordGate.Y_sqrt(qubit1[0]),
+            ),
+        ),
+        ([1, 0], cirq.Circuit(cirq.Y(qubit1[0]), cirq.Y(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.Z(qubit1[0]), cirq.Z(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.H(qubit1[0]), cirq.H(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.S(qubit1[0]), cirq.S(qubit1[0]))),
+        ([0, 1], cirq.Circuit(cirq.X(qubit1[0]), cirq.X(qubit1[0]), cirq.X(qubit1[0]))),
+        ([0, 1], cirq.Circuit(cirq.Y(qubit1[0]), cirq.Y(qubit1[0]), cirq.Y(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.Z(qubit1[0]), cirq.Z(qubit1[0]), cirq.Z(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.S(qubit1[0]), cirq.S(qubit1[0]), cirq.S(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.H(qubit1[0]), cirq.H(qubit1[0]), cirq.S(qubit1[0]))),
+        ([1, 0], cirq.Circuit(cirq.H(qubit1[0]), cirq.H(qubit1[0]), cirq.T(qubit1[0]))),
+        ([0.5, 0.5], cirq.Circuit(cirq.H(qubit1[0]), cirq.H(qubit1[0]), cirq.H(qubit1[0]))),
+        (
+            [0.5, 0.5],
+            cirq.Circuit(
+                cirq.SingleQubitCliffordGate.X_sqrt(qubit1[0]),
+                cirq.SingleQubitCliffordGate.X_sqrt(qubit1[0]),
+                cirq.SingleQubitCliffordGate.X_sqrt(qubit1[0]),
+            ),
+        ),
+        (
+            [0.5, 0.5],
+            cirq.Circuit(
+                cirq.SingleQubitCliffordGate.Y_sqrt(qubit1[0]),
+                cirq.SingleQubitCliffordGate.Y_sqrt(qubit1[0]),
+                cirq.SingleQubitCliffordGate.Y_sqrt(qubit1[0]),
+            ),
+        ),
+        (
+            [0.387577, 0.612423],
+            cirq.Circuit(
+                cirq.Rx(rads=1)(qubit1[0]),
+                cirq.Circuit(cirq.Ry(rads=2)(qubit1[0])),
+                cirq.Circuit(cirq.Rz(rads=1)(qubit1[0])),
+            ),
+        ),
+        (
+            [0.551923, 0.448077],
+            cirq.Circuit(
+                cirq.Rx(rads=1)(qubit1[0]),
+                cirq.Circuit(cirq.Ry(rads=2)(qubit1[0])),
+                cirq.Circuit(cirq.Rx(rads=3)(qubit1[0])),
+            ),
+        ),
+        (
+            [0.257261, 0.742739],
+            cirq.Circuit(
+                cirq.Rx(rads=1)(qubit1[0]),
+                cirq.Circuit(cirq.Rz(rads=2)(qubit1[0])),
+                cirq.Circuit(cirq.Rx(rads=3)(qubit1[0])),
+            ),
+        ),
+    ],
+)
+def test_transpiling_one_qubit_circuits_to_native_gates(ideal_results, circuit):
+    transpiled_circuit = cirq.optimize_for_target_gateset(circuit, gateset=AriaNativeGateset())
+    simulator = cirq.Simulator()
+    result = simulator.simulate(transpiled_circuit)
+    probabilities = np.abs(result.final_state_vector) ** 2
+    np.testing.assert_allclose(probabilities, ideal_results, atol=1e-3)
```

Comment:
The atol=1e-3 is very loose, please replace with 1e-6 or so here and below.

Reply:
Changed to 1e-5, this should suffice since the test values have 6 digit precission